www/src/AdminBundle/Model/ImportTwine.php

```diff
@@ -0,0 +1,592 @@
+<?php
+/* Copyright (C) 2016 Michael Giesler
+ *
+ * This file is part of Dembelo.
+ *
+ * Dembelo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Dembelo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License 3 for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License 3
+ * along with Dembelo. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/**
+ * @package AdminBundle
+ */
+
+namespace AdminBundle\Model;
```

Comment:
Ich bevorzuge weiterhin AdminBundle\Model\ImportTwine.

AdminBundle/Import/... finde ich ungünstig, weil es das Wurzelverzeichnis "zumüllt". Dort liegen derzeit: Command/, Controller/, Document/, Resources/, Tests/  - alles Verzeichnisse, die Symfony-spezifisch sind. Weitere Dembelo-spezifische Klassen/Verzeichnisse sollen da nicht kreuz und quer zwischenliegen.

AdminBundle/Command trifft nicht zu, da die Import-Klasse sowohl vom Command als auch vom Controller aus angesprochen wird.

AdminBundle/UseCase trifft nicht zu, weil es sich nicht um einen UseCase handelt, der in der Klasse implementiert ist.

Ich habe noch die Empfehlungen Service und Utils gefunden - beides nicht so glücklich, wie ich finde.

Der Begriff Model bezieht sich in Symfony nicht ausschließlich auf Datenbankmodels, sondern im Sinne von MVC auf Logik im Allgemeinen, zumal in DembeloMain/Model/Repository/Doctrine in einer tieferen Schicht unterhalb von Model wieder die Assoziation zur Datenbank aufgenommen wird. Für mein Verständnis ist die Model-Schicht genau dafür da: die Logik. Wenn du hier eine andere Auffassung hast, klär mich auf.